execution/gethexec/executionengine.go

```diff
@@ -275,6 +275,59 @@ func (s *ExecutionEngine) SequenceTransactions(header *arbostypes.L1IncomingMess
 	})
 }
 
+func (s *ExecutionEngine) SequenceMessage(msg arbostypes.L1IncomingMessage, hooks *arbos.SequencingHooks) (*types.Block, error) {
```

Comment:
This function is `sequenceTransactionsWithBlockMutex` but without filtering transactions according to the `hooks`.

Reply:
@ImJeremyHe Suggest adding a documentation comment to the function. Also maybe call it `SequenceTransactionsEspresso`?